requirements/min-versions.in

```diff
@@ -1,10 +1,10 @@
 # minimum install requirements
```

Comment:
You don't need to update this file. This file kept the minimal version supported by APIFlask and will only be used in testing.

Reply:
If I don't update this file, the new test will fail.
```python
        rv = client.post(
            '/',
            data={
                'image': '',
            },
            content_type='multipart/form-data'
        )
>       assert rv.status_code == 200
E       assert 422 == 200
E         +422
E         -200

tests/test_fields.py:54: AssertionError
```

Reply:
You can skip this test based on the version of flask-marshmallow. Example: https://github.com/apiflask/apiflask/blob/4992f8efe0c22025f67eadd366891f25e3a152e6/examples/test_examples.py#L118

Reply:
`flask_marshmallow` has no attribute `__version__`. I'll make a pr for it.
```python
AttributeError: module 'flask_marshmallow' has no attribute '__version__'
```

Reply:
Use `importlib.metadata`:

```
>>> from importlib.metadata import version
>>> version('flask')
'2.3.3'
>>> version('flask-marshmallow')
'0.15.0'
```

Reply:
BTW, could you help update the reference of `flask_sqlalchemy.__version__` and change to use `importlib.metadata.version`?

[That variable is deprecated](https://github.com/pallets-eco/flask-sqlalchemy/blob/main/src/flask_sqlalchemy/__init__.py#L12-L26).